java/feature-layer-rendering-mode-scene/src/main/java/com/esri/arcgisruntime/sample/featurelayerrenderingmodescene/MainActivity.java

```diff
@@ -0,0 +1,179 @@
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class MainActivity extends AppCompatActivity {
+
+  private SceneView mSceneViewTop;
+  private SceneView mSceneViewBottom;
+  private Camera mZoomedIn;
+  private Camera mZoomedOut;
+  private Button mZoomButton;
+  private TextView mNavigatingTextView;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    Point zoomedOutPoint = new Point(-118.37, 34.46, SpatialReferences.getWgs84());
+    Point zoomedInPoint = new Point(-118.45, 34.395, SpatialReferences.getWgs84());
+
+    // define viewpoints
+    mZoomedOut = new Camera(zoomedOutPoint, 42000, 0, 0, 0);
+    mZoomedIn = new Camera(zoomedInPoint, 2500, 90, 75, 90);
+
+    // inflate the zoom button
+    mZoomButton = findViewById(R.id.zoomButton);
+
+    // inflate SceneViews from layout
+    mSceneViewTop = findViewById(R.id.sceneViewTop);
+    mSceneViewBottom = findViewById(R.id.sceneViewBottom);
+
+    // inflate navigating text view
+    mNavigatingTextView = findViewById(R.id.isNavigatingTextView);
+    mNavigatingTextView.setVisibility(View.INVISIBLE);
+
+    // create a scene (top) and set it to render all features in static rendering mode
+    ArcGISScene sceneTop = new ArcGISScene();
+    sceneTop.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.STATIC);
+    sceneTop.getLoadSettings().setPreferredPolylineFeatureRenderingMode(FeatureLayer.RenderingMode.STATIC);
+    sceneTop.getLoadSettings().setPreferredPolygonFeatureRenderingMode(FeatureLayer.RenderingMode.STATIC);
+
+    // create a scene (bottom) and set it to render all features in dynamic rendering mode
+    ArcGISScene sceneBottom = new ArcGISScene();
+    sceneBottom.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+    sceneBottom.getLoadSettings().setPreferredPolylineFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+    sceneBottom.getLoadSettings().setPreferredPolygonFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+
+    // create the service feature table
+    ServiceFeatureTable faultServiceFeatureTable = new ServiceFeatureTable(
+        getResources().getString(R.string.energy_geology_feature_service) + "0");
+    ServiceFeatureTable contactsServiceFeatureTable = new ServiceFeatureTable(
+        getResources().getString(R.string.energy_geology_feature_service) + "8");
+    ServiceFeatureTable outcropServiceFeatureTable = new ServiceFeatureTable(
+        getResources().getString(R.string.energy_geology_feature_service) + "9");
+
+    // create the feature layer using the service feature table
+    FeatureLayer faultFeatureLayer = new FeatureLayer(faultServiceFeatureTable);
+    FeatureLayer contactsFeatureLayer = new FeatureLayer(contactsServiceFeatureTable);
+    FeatureLayer outcropFeatureLayer = new FeatureLayer(outcropServiceFeatureTable);
```

Comment:
I think you should check the load status of these layers, they do not always render for me.  

Here is an example where the top layers load: 

![notrendering](https://user-images.githubusercontent.com/1308001/31923281-9722349c-b82e-11e7-9907-3601f2b4b41b.png)

Here nothing loads

![lesrendering](https://user-images.githubusercontent.com/1308001/31923290-a5599b86-b82e-11e7-9ade-f55ac9b7c202.png)